src/Sarif.UnitTests/TestData/SarifVersionOneToCurrentVisitor/ExpectedOutputs/OneRunWithRules.sarif

```diff
@@ -10,30 +10,32 @@
       "results": [
         {
           "ruleId": "C2001",
+          "ruleIndex": 0,
           "message": {
             "arguments": [ "someVariable" ],
             "messageId": "default"
           }
         },
         {
           "ruleId": "C2001",
+          "ruleIndex": 0,
           "message": {
             "arguments": [ "anotherVariable" ],
             "messageId": "default"
           }
         },
         {
-          "ruleId": "C2002-1",
+          "ruleId": "C2002",
           "message": { "text": "Some testing occurred." }
         },
         {
-          "ruleId": "C2003",
+          "ruleIndex": 2,
```

Comment:
>ruleIndex [](start = 11, length = 9)

In the input file, this result has no `ruleId`, and again it was never right to assign a `ruleKey` to a `ruleId` -- so the v2 `result` has no `ruleId` property. On the other hand, the v1 `ruleKey` _did_ point to a real `rule` object, so this v2 `result` _does_ have a `ruleIndex` property.